### PR TITLE
zoom image from python

### PR DESCRIPTION
**Pull request by rijobro**

### CODE REVIEW

File: src/xSTIR/cSTIR/cstir.cpp [hunk 1]
```diff
@@ -962,6 +962,37 @@ void* cSTIR_writeImage(void* ptr_i, const char* filename)
 	CATCH;
 }
 
+extern "C"
+void* cSTIR_ImageData_zoom_image(void* ptr_im, const size_t zooms_ptr_raw, const size_t offsets_in_mm_ptr_raw,
```
Comment (KrisThielemans) [inline src/xSTIR/cSTIR/cstir.cpp]: from the .h,seems that `size_t`->`PTR_FLOAT`

File: src/xSTIR/cSTIR/cstir.cpp [hunk 2]
```diff
@@ -962,6 +962,37 @@ void* cSTIR_writeImage(void* ptr_i, const char* filename)
...
+                                 const size_t new_sizes_ptr_raw, const char *zoom_options)
```
Comment (KrisThielemans) [inline src/xSTIR/cSTIR/cstir.cpp]:
```suggestion
                                 const size_t new_sizes_ptr_raw, const char *const zoom_options)
```

File: src/xSTIR/cSTIR/include/sirf/STIR/stir_data_containers.h [hunk 3]
```diff
@@ -43,6 +43,7 @@ limitations under the License.
 #include "sirf/common/PETImageData.h"
 #include "sirf/STIR/stir_types.h"
 #include "sirf/common/GeometricalInfo.h"
+#include "stir/zoom.h"
```
Comment (KrisThielemans) [inline src/xSTIR/cSTIR/include/sirf/STIR/stir_data_containers.h]:
```suggestion
#include "stir/ZoomOptions.h"
```

File: src/xSTIR/cSTIR/cstir.cpp (hunk 1 above)
Comment (rijobro) [reply src/xSTIR/cSTIR/cstir.cpp]: think it can be anything (if seen `void *` elsewhere) as it goes through a C-style cast.

**Pull request merged**